typings/index.d.ts

```diff
@@ -74,9 +78,9 @@ declare module 'prismarine-nbt'{
     size: number
   }
   export function writeUncompressed(value: NBT, format?: NBTFormat): Buffer;
-  export function parseUncompressed(value: Buffer, format?: NBTFormat): NBT;
+  export function parseUncompressed(value: Buffer, format?: NBTFormat, options?: ParseOptions): NBT;
   
-  export function parse(data: Buffer, nbtType?: NBTFormat): Promise<{parsed: NBT, type: NBTFormat, metadata: Metadata}>;
+  export function parse(data: Buffer, nbtType?: NBTFormat, _?: never, options?: ParseOptions): Promise<{parsed: NBT, type: NBTFormat, metadata: Metadata}>;
```

Comment:
I typed this with a `_` variable because we still need to support the deprecated `callback` parameter. Eventually when the deprecated functions are removed in a major version we could remove it from the typing.